server_api.py

```diff
@@ -1195,14 +1195,13 @@ def read_text_data(text_id):
         return_msg = {}
                 
         with TextDao() as textDao:
-            info = textDao.getIdSysName(Id=text_id)
-        text_file_name = info["systemName"]
-        type_id = info["typeId"]
+            textInfo = textDao.getIdSysName(Id=text_id)
+        type_id = textInfo["typeId"]
 
```

Comment:
Line 1199 type_id is redundant variable, isn't it?

Reply:
Yes, I will remove it.